### ISSUE TITLE
New feature: Get active activity name

### ISSUE THREAD

**Issue by ignaciogs** [pbreault/adb-idea#74]
Hi, iI think that is feature is very interesting and with this command you can get the active activity name on the screen.

adb shell dumpsys activity activities | grep mFocusedActivity | cut -d ' ' -f 6 | grep -o '[^.]*$'

Thanks by this plugin, it's great.

**Comment by EarlOfEgo** [comment]
I disagree, since this command is never changing you can just use your terminal for it. Imo ADB-Idea always works with the current app and operates on this context. 
Don't get me wrong, it's a good idea and I didn't know it exists but I just think it isn't a good fit into the plugin.
